app/views/hyrax/autopopulation/dashboard/work_fetchers/_hyrax_autopopulation_settings_form.html.erb

```diff
@@ -0,0 +1,19 @@
+<%= form_with(url: hyrax_autopopulation.settings_work_fetchers_path, :html => { class: 'form control' }) do |f| %>
+    <div class="panel-body">
+     <div class="form-group">
+       <%=f.fields_for :settings do |ff| %>
+         <%= ff.label :orcid_ids_for_autopopulation %><br>
+         <%= ff.text_field :orcid_list, :value => " ", class: 'form-control', id: "doi_autopopulation_settings" %><br>
```

Comment:
I'm not sure if rubocop parses view files, but you have mixed `'` and `"` in a few of these partials.

Reply:
To be corrected.